### PR TITLE
Add keep-alive support for a more persistent connection

### LINKED ISSUE #18: connection is lost.
can't keep alive conection..

### PR DESCRIPTION
**Pull request by reinier-millo**
I have rebased master branch into develop, and added support to prevent websocket be closed after 30 seconds. It must solve issue #18

### CODE REVIEW

File: kyros/websocket.py [hunk 1]
```diff
@@ -142,10 +154,16 @@ async def connect(self) -> None:
         listener."""
         logger.debug("Connecting to ws server")
         self.websocket = await websockets.connect(
-            constants.WEBSOCKET_URI, origin=constants.WEBSOCKET_ORIGIN)
+            constants.WEBSOCKET_URI, origin=constants.WEBSOCKET_ORIGIN, close_timeout = None,ping_interval = None)
         logger.debug("Websocket connected")
         self._start_receiver()
 
+    async def keepAlive(self):
```
Comment (lc-at) [inline kyros/websocket.py]: I think that the function name should comply to the PEP-8 standard. There should be docstrings too, I guess.

File: kyros/websocket.py [hunk 2]
```diff
@@ -142,10 +154,16 @@ async def connect(self) -> None:
         listener."""
         logger.debug("Connecting to ws server")
         self.websocket = await websockets.connect(
-            constants.WEBSOCKET_URI, origin=constants.WEBSOCKET_ORIGIN)
+            constants.WEBSOCKET_URI, origin=constants.WEBSOCKET_ORIGIN, close_timeout = None,ping_interval = None)
```
Comment (lc-at) [inline kyros/websocket.py]: The comma needs an extra space.

**Review by lc-at — CHANGES_REQUESTED** [review]
At a glance, it looks quite good to me. However, you need to make sure that your code is consistent with the whole code base (make sure it complies to PEP8).

I haven't tested the code though, but it looks promising.

**Comment by lc-at** [comment]
@rijumone, can you test and review this?

File: kyros/websocket.py (hunk 1 above)
Comment (reinier-millo) [reply kyros/websocket.py]: Sorry, it's my mistake, i have moved out solution from a project code that i'm working. I have uploaded the fix.

File: kyros/websocket.py (hunk 2 above)
Comment (reinier-millo) [reply kyros/websocket.py]: Fixed

**Comment by rijumone** [comment]
> @rijumone, can you test and review this?

@p4kl0nc4t, I would love to. However, I am yet to get this library running on my dev environment owing to #19 . Would it be possible to hop on a Zoom call or something to get me up to speed on this? An overall walk-through of the library would also be greatly appreciated.

**Pull request merged**